### ISSUE TITLE
Do Quick/Nimble support iOS 7?

### ISSUE THREAD

**Issue by modocache** [Quick/Quick#206]
I received the following email:

I am trying to use them both on an Obj-C project targeting iOS 7 and up. After I include the frameworks and add them to the build-phases/Link-binary-with-libraries, my build fails with the following error:

```
ld: embedded dylibs/frameworks are only supported on iOS 8.0 and later (@rpath/Nimble.framework/Nimble) for architecture x86_64
clang: error: linker command failed with exit code 1
```

It seems Quick & Nimble are embedded/dynamic frameworks and they do not work on iOS 7.
I also read somewhere that it is impossible to create static frameworks from swift code.

So, Is there some way to use Quick/Nimble for ObjC/iOS7? do you know if someone succeed in that path?

**Comment by marciok** [comment]
If somebody ended up here (like did) this worked for me:
1. Change ```IPHONEOS_DEPLOYMENT_TARGET ``` value to ```7.0``` on *(Quick/Nimble).xcodeproj/project.pbxproj*
2.  Write a passing test to avoid the: ```IDEBundleInjection.c: Error 3587 loading bundle...``` error
3. Profit! - it should work.

If it didn't work remove Quick/Nimble from your project, add them again, clean and try to build again.

**Comment by modocache** [comment]
@marciok Awesome! Could you submit a pull request with the `IPHONEOS_DEPLOYMENT_TARGET` changes? :pray:

**Comment by jeffh** [comment]
This was merged in, closing. Thanks everyone! :+1:

**Comment by marciok** [comment]
👍😊😊😊

**Comment by johnryan** [comment]
would this be part of a 0.3.1 release?...just deciding whether I should wait for that or make my own fork.

**Comment by modocache** [comment]
@johnryan These changes are included in Quick v0.2.3 (supports Swift 1.1). If someone picks the commits and issues a PR for the master branch, I can release Quick v0.3.1 to include those changes in a Swift 1.2-compatible version! :+1:

**Comment by johnryan** [comment]
@modocache Thanks for the clarification...in that case I'm a little confused, in my podfile looks like:

```
target 'MyTests' do
  pod 'Quick'
  pod 'Nimble'
end
```

but I still get the error: 
[!] The platform of the target `Truly Tests` (iOS 7.0) is not compatible with `Quick (0.3.0)` which has a minimum requirement of iOS 8.0 - OS X 10.10.
